1-introduction.md

```diff
@@ -0,0 +1,60 @@
+
+That's a lot to digest. More simply, Plasma is a method of securely scaling a blockchain. It's not a framework like _Ruby on Rails_, _Express.js_, or _UIKit_. I think about it as a proposed [design pattern](https://en.wikipedia.org/wiki/Software_design_pattern) to allow a blockchain to represent a lot more state updates (transactions).
+
+
+## Under Construction
+
+Plasma is currently under heavy research and development by OmiseGO and multiple teams around the world. The white paper specifies many concepts that are yet to be explored for implementation. In the [Specifications](specifications) section, you can see more concrete specifications of Plasma, as well as implementations of those specs.
+
+
+## Structure
+
+In the most general case, Plasma describes the interaction of a tree of blockchains.
+
+[ XXX - add graphic ]
+
+Going from the leaf chains to the root of the tree, each chain periodically submits a summary update of its state to the parent chain. Eventually, the update of all the updates get submitted to a root chain (e.g. Ethereum). Conceptually, this is much like MapReduce. We map the update procedure through the tree and repeatedly reduce the results.
+
+By doing this, we extend the security of the root chain to the entire structure of blockchains. The blockchains that are the children of the root can operate concurrently and :boom:, billions of transactions per second. Easy!
+
+I know I'm being pretty hand wavy about all of this right now. Don't worry, I'll get into detail in the following posts. All you really need to know right now is...
+
+[ XXX - yo dawg graphic ]
+
+
+## Root Chain
+
+Plasma is blockchain agnostic for its root chain. The purpose of the root chain is to secure the entire Plasma structure. It's a design pattern that can be implemented on any blockchain that can support the interaction between a first-level child chain and the root chain.
+
+Given that the implementation and design details of Plasma are currently under active research by many teams, the protocol between a child chain and root chain is in flux. Since it's in flux, we need a root chain that can support a changing protocol and that means using a root chain that can support smart contracts.
+
+At OmiseGO and for the rest of these posts, we'll be using Ethereum as our root chain. That means we'll be using Solidity as our smart contract language in the examples.
+
+
+## Plasma Chain
+
+All the blockchains that are children of the root are called _Plasma chains_. Each Plasma chain will operate as a child and potentially as a parent too, depending on whether it's a an internal node or leaf in the tree.
+
+### As a child chain
+
+The Plasma chain acts as a child by periodically reporting an update back to its parent chain (which can also the be the root chain). This magical update up to the parent chain takes the form of the _Merkle root of a Merkle tree of transactions_ from the most recent block of the child chain.
```

Comment:
A merkle root of a merkle tree of transactions is fairly spec/implementation specific, though it's what we're currently focused on.

Reply:
Which part? Would you feel better about saying `state transitions`?